src/Components/Notifications/Notification.tsx

```diff
@@ -23,33 +26,28 @@ import createLogger from "Utils/logger"
 import { NotificationErrorMessage } from "Components/Notifications/NotificationErrorMessage"
 import { useClientQuery } from "Utils/Hooks/useClientQuery"
 import { PartnerShowOpenedNotification } from "Components/Notifications/PartnerShowOpenedNotification"
-import { DESKTOP_HEIGHT } from "Apps/Notifications/notificationsutils"
+import { NotificationCollectorProfileUpdatePromptFragmentContainer } from "Components/Notifications/NotificationCollectorProfileUpdatePrompt"
 
 const logger = createLogger("NotificationItem")
 
+// FIXME: Should probably be a column span
 export const NOTIFICATION_MAX_WIDTH = 600
-export const SUPPORTED_NOTIFICATION_TYPES = [
+export const SUPPORTED_NOTIFICATION_TYPES: NotificationTypesEnum[] = [
```

Comment:
Relay could refactor their compiler to generate literal types instead of enums 😞

Reply:
@mzikherman added the `item` type. There's a lot wrong with this. Like: why is there a redirect buried inside of a sub-sub-sub-component (took me like an hour to dig this out). And this isn't even a good way to handle this. 😭